jdk/src/java.base/share/classes/java/util/Arrays.java

```diff
@@ -83,6 +90,26 @@
     // Suppresses default constructor, ensuring non-instantiability.
     private Arrays() {}
 
+    private static final class GPUAssistHolder {
```

Comment:
Please promote this to a package-private class to reduce the potential for merge conflicts.